doc/charm++/manual.rst

```diff
@@ -8711,6 +8711,38 @@ computational power because those dedicated cores are not utilized at
 all during most of the application's execution time. This case indicates
 the necessity of a unified runtime supporting both types of parallelism.
 
+
+Tass with StealQ for within-node load balancing
```

Comment:
Typo: "Task". And maybe better as "Stealable tasks for within-node load balancing"